### DISCUSSION TITLE
High CPU Usage After Codespaces Stopped

### DISCUSSION THREAD

**Discussion by braedonondos** [community/community#75687] [Codespaces]
### Select Topic Area

Bug

### Body

MacOS Sonoma 14.1.1
Codespaces v1.16.2

When using the codespaces extension on VSCode to access a codespace, if I leave the codespace open until it automatically stops the codespace (bringing the popup window that says something like "[codespace name] has been stopped" and has the option to reconnect), the codespaces extension will begin using all of the CPU on my laptop as shown in Activity Monitor, causing my laptop to heat up and the battery to drain very quickly. I have reproduced this issue multiple times with this same scenario: just simply connecting to a codespace and letting it stop automatically.

**Comment by bhildebay** [comment]
I confirm, same behavior on my machine as well. (Windows 10, Intel, VSCode 1.84.2, Codespaces Extension 1.16.2)
